packages/itwinui-css/src/table/table.scss

```diff
@@ -2,7 +2,13 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 @import '../icon/index';
-@import './variables';
+
+$iui-table-density-vertical: $iui-baseline * 1.5;
```

Comment:
What is the idea of moving variables here? Better visibility?

Reply:
This change is just for consistency. Other components such as Badge, Color Picker, Date Picker, Menu, Slider, Toggle Switch, and Tree list their variables at the top of the file instead of having a separate file for them.